ansible_rulebook/websocket.py

```
@@ -81,6 +82,10 @@ async def _connect_websocket(
     **kwargs: list,
 ) -> tp.Any:
     logger.info("websocket %s", settings.websocket_url)
+    if not validate_url(settings.websocket_url, "websocket"):
+        logger.error("Invalid websocket url.")
+        return 1
```

Would it be better to raise an error here instead?